thunder/executors/nvfuserex_impl.py

```diff
@@ -412,6 +482,14 @@ def compute_contiguity(
     return tuple(tuple(x) for x in nv_compute_td(shape, stride))
 
 
+def make_key_from_dtensor(dtensor: DTensor) -> tuple[str]:
```

Comment:
According to the current type hint, the else branch is never taken.
```suggestion
def make_key_from_dtensor(dtensor: DTensor | Any) -> tuple[str]:
```